flutter/lib/src/binding_wrapper.dart

```diff
@@ -73,37 +73,61 @@ typedef FrameTimingCallback = void Function(
     DateTime startTimestamp, DateTime endTimestamp);
 
 mixin SentryWidgetsBindingMixin on WidgetsBinding {
-  DateTime? _startTimestamp;
-  FrameTimingCallback? _frameTimingCallback;
+  FrameTimingCallback? _onDelayedFrame;
   ClockProvider? _clock;
-
+  Stopwatch? _stopwatch;
+  Duration? _expectedFrameDuration;
+  bool _isTrackingActive = false;
   SentryOptions get _options => Sentry.currentHub.options;
 
   @internal
-  void registerFramesTracking(
-      FrameTimingCallback callback, ClockProvider clock) {
-    _frameTimingCallback ??= callback;
+  void initializeFramesTracking(
+      FrameTimingCallback onDelayedFrame,
+      ClockProvider clock,
```

Comment:
We can just use `_options.clock`, no need to pass it here.

Reply:
Changed it to inject options instead, I need the clock to be injectable somehow due to tests